samples/VSAndroidApp/VSAndroidApp.csproj

```diff
@@ -25,7 +25,7 @@
     <MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
     <AndroidEnableSGenConcurrent>true</AndroidEnableSGenConcurrent>
     <AndroidUseAapt2>true</AndroidUseAapt2>
-    <AndroidHttpClientHandlerType>Xamarin.Android.Net.AndroidClientHandler</AndroidHttpClientHandlerType>
+    <AndroidHttpClientHandlerType>Xamarin.Android.Net.AndroidMessageHandler</AndroidHttpClientHandlerType>
```

Comment:
This sample is meant to be the default template you get in VS for Xamarin.Android, not .NET 6.

So some concerns are:
1. Do we want this the default in Xamarin.Android? We'll have to update templates for this new value, they are in a different repo.
2. We'd also need the Xamarin extension in Visual Studio to update its UI on the properties pages.
3. Should we change the default value of `$(AndroidHttpClientHandlerType)` when it is blank?

These lead me to think we should make these changes in .NET 6 only and not affect "classic" Xamarin.Android?

Reply:
Yeah, I believe this change should only be for .NET6

Reply:
Ok @simonrozsival, if you want to follow what we did elsewhere is use `#if MONOANDROID1_0` for Xamarin.Android, and `#if !MONOANDROID1_0` for .NET 6.

We could also exclude whole files from the compilation, but that probably won't apply here.